numpy/core/src/umath/override.c

```diff
@@ -257,8 +257,8 @@ normalize_reduce_args(PyUFuncObject *ufunc, PyObject *args,
             }
             obj = PyTuple_GetSlice(args, 3, 4);
         }
-        /* Remove initial=np._NoValue */
-        if (i == 5 && obj == NoValue) {
+        /* Remove {initial,where}=np._NoValue */
+        if (i >= 5 && obj == NoValue) {
```

Comment:
Why is `where=np._NoValue` even allowed?

Reply:
It gets used by `sum` etc in `fromnumeric`. In principle, though, those also strip it away again (unless overridden with `__array_function__`). Unlike `initial`, it does not get used in `_methods`, so I think it can indeed be removed. Will do so.